### DISCUSSION TITLE
Support skip changelog

### DISCUSSION THREAD

**Discussion by RiverLoveMU** [changesets/changesets#865] [Ideas]
Is it possible to provide a method to skip changelog like standard-version?
<img width="877" alt="image" src="https://user-images.githubusercontent.com/28769069/178245493-4e3b0599-4018-4182-a8ac-3971892bd740.png">

**Comment by CobyPear** [comment]
I am also curious about this. Is it currently supported? Is there a way to skip the changelog generation only in prerelease versions? Thanks!

**Comment by inthegreenwild** [comment]
You can set "changelog" to false in the config file. It would be nice to have this option as a cli flag though
